python/py-distributed/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-distributed
-version             2.3.0
+version             2.3.2
```

Comment:
This commit is unrelated and you need to get it out of your pull request.

Reply:
I hope that commit a8170a9 did the trick.